typings/index.d.ts

```diff
@@ -1,95 +1,61 @@
-// Type definitions for CodeceptJS 1.0.2
+// Type definitions for CodeceptJS 2.1.4
```

Comment:
Is this file still actively used internally?, as I don't see any indication it could have used externally. If used internally, I will update it accordingly so it will support the same code

Reply:
Used and updated, except this line. But I think it should be generated by package.json version

Reply:
or removed